src/screens/OrganizationDashboard/OrganizationDashboard.tsx

```diff
@@ -190,16 +195,19 @@ function organizationDashboard(): JSX.Element {
                         <h6>No upcoming events</h6>
                       </div>
                     ) : (
-                      upcomingEvents.slice(0, 5).map((event) => {
-                        return (
-                          <CardItem
-                            type="Event"
-                            key={event._id}
-                            time={event.startDate}
-                            title={event.title}
-                          />
-                        );
-                      })
+                      sortAndSliceEventsByStartDate(upcomingEvents).map(
```

Comment:
Why are events sorted here on the client-side?

You can use `eventsByOrganizationConnection` query to fetch posts from backend itself sorted by `startDate`.